restapi/src/main/java/io/stargate/web/docsapi/models/DocCollection.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.web.docsapi.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DocCollection {
+  @JsonProperty("name")
+  private String name;
+
+  @JsonProperty("upgradeAvailable")
+  private boolean upgradeAvailable;
+
+  @JsonProperty("upgradeType")
+  private CollectionUpgradeType upgradeType;
+
+  @ApiModelProperty(required = true, value = "The name of the collection.")
+  @JsonProperty("name")
+  public String getName() {
+    return name;
+  }
+
+  @ApiModelProperty(
+      required = true,
```

Comment:
Should this be required? Will the user know?

Reply:
oh wait this is wrong yea

Will change